src/globals.d.ts

```diff
@@ -1,3 +1,4 @@
 declare module "@elastic/react-search-ui";
 declare module "@elastic/react-search-ui-views";
 declare module "@elastic/search-ui-app-search-connector";
+declare module "@react-show-more-text";
```

Comment:
There shouldn't be a need for this since types have already been declared for the library. Can remove.